### PR TITLE
Added FAQ section to team page

### PR DESCRIPTION
**Pull request by EinarBalan**
Ripped directly from hoth website ty

lmk if more questions should be added/answers should be tweaked.

### CODE REVIEW

File: src/components/PhotoPage/FAQSection.js
Comment (khxia) [inline src/components/PhotoPage/FAQSection.js:17]: let's remove this LMAO

**Review by khxia — CHANGES_REQUESTED** [review]

File: src/components/PhotoPage/FAQSection.js
Comment (TimothyGu) [reply src/components/PhotoPage/FAQSection.js:135]: Also, please use relative URLs. This way during development we can visit the archive page belonging to the local instance.

```suggestion
					Check out the <Link href="/archive">archive</Link> page!
```

File: src/components/PhotoPage/FAQSection.js
Comment (TimothyGu) [inline src/components/PhotoPage/FAQSection.js:128]:
```suggestion
					We recommend attending workshops and events (such as HOTH or Hack
					Kitchen), joining a Hack Fam, or asking us questions in the ACM discord!
```

Currently there's a space rendered before the comma.

File: src/components/PhotoPage/FAQSection.js
Comment (TimothyGu) [inline src/components/PhotoPage/FAQSection.js:143]: Same here
```suggestion
					We start recruitment for internship positions every fall,
					following the ACM general meeting (which you should attend)!
```

File: src/components/PhotoPage/FAQSection.js
Comment (jodymlin) [reply src/components/PhotoPage/FAQSection.js:128]: We might not always be holding events Hack Fam, HOTH, or Hack Kitchen every quarter. To keep maintaining this simple we should just saying something like "check out our events page to see what events we have this quarter! Everyone is able to join!" Be sure to emphasize that _anyone_ can join. That's something we always try to let people know.

File: src/components/PhotoPage/FAQSection.js
Comment (jodymlin) [inline src/components/PhotoPage/FAQSection.js:144]:
```suggestion
					After a quarter of shadowing officers, interns may have the opportunity to become full-time officers. All of our current officers started as interns and we highly encourage you to look out for applications in the Fall!
```
Or something similar to that 😃 

For "legal" reasons we shouldn't guarantee that all interns become officers. While historically all interns have become officers, this has never actually been a guarantee.

File: src/components/PhotoPage/FAQSection.js
Comment (jodymlin) [inline src/components/PhotoPage/FAQSection.js:151]:
```suggestion
					Send us an email at <Link href='mailto:uclaacmhack@gmail.com'>uclaacmhack@gmail.com</Link>!
```

**Review by jodymlin — CHANGES_REQUESTED** [review]

**Comment by khxia** [comment]
Let's make the "Team" section on the navigation tab be named "About" instead.

**Comment by khxia** [comment]
This looks good to me. But we decided in the last meeting that we should talk a little more about where to put the FAQ, so I'll wait until then to approve it.

**Review by khxia — APPROVED** [review]

**Pull request merged**
